src/core/thread/key_manager.cpp

```diff
@@ -62,8 +62,10 @@ const uint8_t KeyManager::kTrelInfoString[] = {'T', 'h', 'r', 'e', 'a', 'd', 'O'
 
 void SecurityPolicy::SetToDefault(void)
 {
+    Clear();
     mRotationTime = kDefaultKeyRotationTime;
     SetToDefaultFlags();
+
```

Comment:
Remove extra line to satisfy pretty check.
```suggestion
    mRotationTime = kDefaultKeyRotationTime;
    SetToDefaultFlags();
```